card/mod/pointer/lib/javascript/script_pointer_list_editor.js.coffee

```diff
@@ -0,0 +1,48 @@
+$(window).ready ->
+  # add pointer item when clicking on "add another" button
+  $('body').on 'click', '.pointer-item-add', (event)->
+    decko.addPointerItem this
+    event.preventDefault() # Prevent link from following its href
```

Comment:
is this what you said is broken now?

Reply:
oh, in a ticket that is already finished: https://www.pivotaltracker.com/story/show/157551476

Reply:
the problem was that the call for disabling the button was on window ready and not slot ready